src/lib/tabs/tab-nav-bar/tab-nav-bar.spec.ts

```diff
@@ -273,6 +274,16 @@ describe('MatTabNavBar', () => {
 
     expect(tabLink.tabIndex).toBe(3, 'Expected the tabIndex to be have been set to 3.');
   });
+
+  it('should set role on tablist and tab', () => {
+    const fixture = TestBed.createComponent(TabLink);
+    fixture.detectChanges();
+
+    const tabList = fixture.debugElement.query(By.css('[role=tablist]'));
```

Comment:
You query by `role=tablist` which means the role must be tablist. Is it better to query by `class=mat-tab-links` and check the role?

Reply:
Done.